db/schema.rb

```diff
@@ -1147,12 +1147,12 @@
     t.index ["name"], name: "index_tags_on_name", unique: true
   end
 
-  create_table "terms_of_service_files", force: :cascade do |t|
-    t.datetime "created_at",              null: false
-    t.datetime "updated_at",              null: false
-    t.string   "attachment_file_name"
-    t.string   "attachment_content_type"
-    t.integer  "attachment_file_size"
+  create_table "terms_of_service_files", id: :serial, force: :cascade do |t|
```

Comment:
I double-checked this and I don't know where this `id: serial` comes from. Your database seems to have introduced it. Can you try the following?
```
git fetch upstream master:master
git checkout master
./bin/rails db:reset
git checkout fit/schema
./bin/rails db:migrate
```

That should be the right way to regenerate the schema. Do you still end up with the same result?

Reply:
Hey @mkllnk sorry for the late reply.

Maybe I ran the migration in some branch still waiting for integration?

Anyway here is what I did following your suggestion:
```
git checkout master
git fetch upstream master:master
./bin/rails db:reset
git branch -d fit/schema
git checkout fit/schema
bundle install
git merge master
./bin/rails db:migrate
git add db
git commit -v
git push
```